src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,362 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level P-ERK time integration schemes
+abstract type PERK end
+# Abstract base type for single/standalone P-ERK time integration schemes
+abstract type PERKSingle <: PERK end
+
+function compute_a_coeffs(num_stage_evals, se_factors, mon_coeffs)
+    a_coeffs = mon_coeffs
+
+    for stage in 1:(num_stage_evals - 2)
+        a_coeffs[stage] /= se_factors[stage]
+        for prev_stage in 1:(stage - 1)
+            a_coeffs[stage] /= a_coeffs[prev_stage]
+        end
+    end
+
+    return reverse(a_coeffs)
+end
+
+function compute_PERK2_butcher_tableau(num_stages, eig_vals, tspan,
+                                       bS, c_end)
+
+    # c Vector form Butcher Tableau (defines timestep per stage)
+    c = zeros(num_stages)
+    for k in 2:num_stages
+        c[k] = c_end * (k - 1) / (num_stages - 1)
+    end
+    se_factors = bS * reverse(c[2:(end - 1)])
```

Comment:
Maybe we also rename `se_factors` file-wide to `bc_factors` as this is semantically more close to their actual origin.